v0/destinations/airship/transform.js

```diff
@@ -0,0 +1,119 @@
+const btoa = require("btoa");
+const { EventType } = require("../../../constants");
+
+const {
+  CONFIG_CATEGORIES,
+  MAPPING_CONFIG,
+  BASE_URL_EU,
+  BASE_URL_US
+} = require("./config");
+
+const {
+  constructPayload,
+  defaultPostRequestConfig,
+  removeUndefinedAndNullValues,
+  defaultRequestConfig,
+  CustomError
+} = require("../../util");
+
+function responseBuilderSimple(message, category, destination) {
```

Comment:
Let's segregate this into separate functions for Identify, Track, Group. It will be easy in terms of code readability and to classify the cases.

Reply:
Done.